scripts/tools/silabs/retrieve_sl_commits.py

```diff
@@ -41,7 +43,7 @@ def main():
     [PREFIX] commits:
     <full_commit_sha> -- <Commit_Title>
     """,
-    epilog= """
+                                     epilog="""
```

Comment:
I think this is a typo

Reply:
@rcasallas-silabs @lpbeliveau-silabs It's not a typo - at least for the restyler.
epilog is an function argument so the restyler alligned it with the function arguemnts

Reply:
Restyler does what restyler does...